includes/amp-helper-functions.php

```diff
@@ -931,6 +931,7 @@ function amp_get_content_sanitizers( $post = null ) {
 		'AMP_Style_Sanitizer'             => [
 			'include_manifest_comment' => ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ? 'always' : 'when_excessive',
 		],
+		'AMP_Meta_Sanitizer'              => [],
```

Comment:
This needs to be added below as well, under `Force style sanitizer and whitelist sanitizer to be at end.`

Reply:
Resolved in https://github.com/ampproject/amp-wp/pull/3758/commits/af541751fc9706fceb9de77c0ef339b40cd4b08c